liquidity-sources/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -186,9 +278,83 @@ async fn update_recently_used_outdated_pools(
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+    use serde_json::json;
     use std::str::FromStr;
 
-    use super::*;
+    #[test]
+    fn encode_pool_info() {
+        assert_eq!(
+            json!({
+                "address": "0x0001fcbba8eb491c3ccfeddc5a5caba1a98c4c28",
+                "tokens": [
+                    {
+                        "id": "0xbef81556ef066ec840a540595c8d12f516b6378f",
+                        "symbol": "BCZ",
+                        "decimals": "18",
+                    },
+                    {
+                        "id": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+                        "symbol": "WETH",
+                        "decimals": "18",
+                    }
+                ],
+                "state": {
+                    "sqrt_price": "792216481398733702759960397",
+                    "liquidity": "303015134493562686441",
+                    "tick": "-92110",
+                    "liquidity_net": {
+                        "ticks": [],
+                    },
+                    "fee": "10000",
+                },
+                "gas_stats": {
+                    "mean_gas": "300000",
```

Comment:
Not sure if this is on purpose but I just called it "mean" here.

Reply:
Added renaming on serialization... just wanted to name it `mean_gas` for clarity.